File: benchmark/fibonacci_recursive/fib.py
def fib(n):
    if n < 2:
        return n
    return fib(n - 1) + fib(n - 2)


def main():
    n = 40
    result = fib(n)
    print(f"Fib({n}) = {result}")


if __name__ == "__main__":
    main()

File: benchmark/fibonacci_recursive/fib.c
#include <stdio.h>

long long fib(long long n) {
    if (n < 2) return n;
    return fib(n - 1) + fib(n - 2);
}

int main() {
    long long n = 40;
    long long result = fib(n);
    printf("Fib(%lld) = %lld\n", n, result);
    return 0;
}
